src/fidesops/schemas/dataset.py

```diff
@@ -62,14 +62,18 @@ class FidesCollectionKey(ConstrainedStr):
     def validate(cls, value: str) -> str:
 
         values = value.split(".")
-        if (
-            len(values) == 2
-            and FidesKey.validate(values[0])
-            and FidesKey.validate(values[0])
-        ):
-            return value
-
-        raise FidesValidationError(
+        try:
+            if (
+                len(values) == 2
+                and FidesOpsKey.validate(values[0])
+                and FidesOpsKey.validate(values[0])
```

Comment:
this was already like this, but this looks like a bug, we should be checking the second piece too [1]